spec/music_album_spec.rb

```diff
@@ -0,0 +1,16 @@
+require_relative '../classes/music_album'
+
+describe MusicAlbum do
+  context 'It should create a musicAlbum instance with onspotify, name and published date' do
+    music = MusicAlbum.new(publish_date: '2012-05-01', name: true, on_spotify: 'Hello')
+    it 'creates new musicAlbum with the given parameters' do
+      expect(music).to be_an_instance_of MusicAlbum
+    end
+    it 'should check if it can be archived' do
+      expect(music.on_spotify).to eq 'Hello'
+    end
```

Comment:
This does not check whether music is archived.
Please add the correct method to check whether the item is archived

Reply:
You also need to check both cases, when the music album is archived and when it is not archived